spec/controllers/v1/actions_controller_spec.rb

```diff
@@ -3,22 +3,19 @@
 RSpec.describe V1::ActionsController, type: :controller do
   describe "#create" do
     before do
-      FactoryGirl.create(:activity, template_id: 'foo')
+      FactoryGirl.create(:activity, template_id: 'real_id')
       FactoryGirl.create(:person, phone: '510-555-4444', email: 'joe@example.com')
     end
-    let (:person) {  }
-    it "creates action based on phone" do
-      post :create, phone: '510-555-4444', template_id: 'foo'
-      expect(JSON.parse(response.body)['success']).to be true
-    end
-    it "creates action based on email" do
-      post :create, email: 'joe@example.com', template_id: 'foo'
+    it "creates action with good parameters" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.